src/schema/server_settings.py

```diff
@@ -324,12 +326,16 @@ class ServerSettings(BaseModel):
         label="Story LLM Temperature",
         description="Temperature (creativity-factor) for the narrative generation. 0=Robot, 1=Bonkers, 0.4=Default",
         type="float",
+        min=0,
+        max=1,
```

Comment:
i'm sure we don't really care, but with OpenAI, you can set temperature up to 2.